x-pack/plugins/apm/kibana.json

```diff
@@ -16,7 +16,8 @@
     "licensing",
     "observability",
     "ruleRegistry",
-    "triggersActionsUi"
+    "triggersActionsUi",
+    "share"
```

Comment:
Based on [this discussion](https://elastic.slack.com/archives/C877NKGBY/p1648740259977729) it makes more sense to have the plugin as `required plugin` instead of having a guard around it.